### DISCUSSION TITLE
TriCaster tally API update (from TCP on port 5951 to Websocket on port 80)

### DISCUSSION THREAD

**Discussion by kanelp** [josephdadams/TallyArbiter#728] [General]
I'm noticing that the API interface used by TallyArbiter is the older TCP/5951 interface.  This provides basic tally information.

If the code was updated to use the new tally API (accessible via change_notification Websocket connection port 80), it would provide more accurate tally information.  For example, if a M/E is select on program or preview output, the tally API will tell you all of the individual sources that are currently tallied, not just that a M/E is on program or preview.

**Comment by josephdadams** [comment]
submit a PR, @kanelp :)
